eng/targets/Helix.Common.props

```diff
@@ -27,4 +27,21 @@
 
     <!--  TODO: re-enable Debian.9.Arm64.Open and Ubuntu.1804.Arm64.Open    -->
   </ItemGroup>
+
+  <ItemGroup Condition="'$(IsWindowsOnlyTest)' == 'true'">
+    <HelixAvailablePlatform Include="Windows" />
+
+    <HelixProjectTargetQueue  Include="Windows.10.Amd64.EnterpriseRS3.ASPNET.Open" />
+    <HelixProjectTargetQueue  Include="Windows.81.Amd64.Open" />
+    <!-- Disabled until https://github.com/dotnet/core-eng/issues/5506 is resolved -->
+    <!-- <HelixProjectTargetQueue Condition="'$(IsWindowsOnlyTest)' == 'true'" Include="Windows.7.Amd64.Open" /> -->
+
+    <HelixContent Include="..\..\..\tools\update_schema.ps1" />
```

Comment:
Let's make these absolute paths, otherwise this will be treated as relative to the project directory. This is a common props file, so project directories are unlikely to all have the same relative path to these files.